src/protocol/mdns.rs

```diff
@@ -128,9 +128,8 @@ impl Mdns {
     ) -> crate::Result<Self> {
         let socket = Socket::new(Domain::IPV4, Type::DGRAM, Some(Protocol::UDP))?;
         socket.set_reuse_address(true)?;
-        // TODO: fix
-        // #[cfg(unix)]
-        // socket.set_reuse_port(true)?;
+        #[cfg(unix)]
+        socket.set_reuse_port(true)?;
```

Comment:
dq: What was the reason this was commented out in the past with a "todo: fix"? Did something break previously?

Reply:
AFAIR when I wrote the code, I was using an incorrect version of `socket2` or something which caused compilation to fail on Linux even though I had `#[cfg(unix)]`. Since the code worked without `port_reuse(true)`, I just commented the code out and was planning on revisiting it in the future but never did until now.